src/app/components/media/MediaCardLarge.module.css

```diff
@@ -9,13 +9,10 @@
   > .video-media-card,
   > .webpage-media-card {
     flex: 1 1;
+    min-height: calc(226px * (16/9));
     min-width: calc(200px * (16/9));
   }
 
-  .aspect-ratio-media-card {
```

Comment:
removing this is the actual fix. The min-height above is just to be extra careful